qa/L0_perf_resnet/test.sh

```diff
@@ -67,9 +67,10 @@ INSTANCE_CNT=1
 CONCURRENCY=1
 
 # Disable TF-TRT test on Jetson due to Segfault
+# Disable ORT-TRT test on Jetson due to support being disabled
```

Comment:
Why is the support disabled? Is there any plans to enable it in the future?

Reply:
It was disabled in 22.03. We're waiting on Microsoft to confirm the same